gtest-parallel

```diff
@@ -346,6 +358,7 @@ if options.format == 'raw':
   pass
 elif options.format == 'filter':
   logger = FilterFormat()
+  logger.zip_file = os.path.join(options.output_dir, 'passing_test_logs.zip')
```

Comment:
Let's have one for everything, then separate logs for failed and interrupted. How about this style:
modules_unittests.log (contains all logs)
modules_unittests_FAILED.log (only failed)
modules_unittests_INTERRUPTED.log (only interrupted).